src/commands/deleteContainerizedFunctionApp/DeleteContainerizedFunctionAppStep.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { createWebSiteClient } from "@microsoft/vscode-azext-azureappservice";
+import { AzureWizardExecuteStep, nonNullValueAndProp } from "@microsoft/vscode-azext-utils";
+import { ProgressLocation, window } from "vscode";
+import { ext } from "../../extensionVariables";
+import { localize } from "../../localize";
+import { type DeleteFunctionappWizardContext } from "./DeleteFunctionAppWizardContext";
+
+export class DeleteContainerizedFunctionappStep extends AzureWizardExecuteStep<DeleteFunctionappWizardContext> {
+    public priority: number = 100;
+
+    public async execute(context: DeleteFunctionappWizardContext): Promise<void> {
+        const deleting: string = localize('DeletingFunctionApp', 'Deleting function app "{0}"...', context.site.name);
+        const deleteSucceeded: string = localize('DeleteFunctionAppSucceeded', 'Successfully deleted function app "{0}".', context.site.name);
+
+        await window.withProgress({ location: ProgressLocation.Notification, title: deleting }, async (): Promise<void> => {
```

Comment:
Question, can we skip showing this `withProgress` since we already have the live update showing in the activity log?

Reply:
Yeah, I think so. It should be our pattern going forward-- maybe a notification that says:
`Deleting function "{0}". To check progress, view Azure Activity Log.` with a button to take them there?

Reply:
Double check me on this, but it seems like anytime the activity log is activated through the wizard, the activity log pane automatically gets focus so it'd probably be hard for the user to miss.

Reply:
Oh, you mean tree item description, nevermind, yeah that makes perfect sense.

Reply:
Actually, I didn't realize that the Azure activity log was already focused whenever something happened there so maybe just disregard what I said.

The tree item description might be a bit too verbose to show it there.